src/Query.sql

```diff
@@ -75,38 +71,70 @@ Query- select SNAME from Salespeople where SNUM IN (select SNUM from Orders);
        | Riffikn |
        +---------+
 
-
 6) List names of all customers matched with the salespeople serving them.
 --------------------------------------------------------------------------
-Query- select CNAME from Customers where SNUM IN (select SNUM from Salespeople);
-       Result Empty.
-
+Query- SELECT CNAME FROM customers WHERE SNUM IN (SELECT SNUM FROM salespeople);
+       +----------+
+       | CNAME    |
+       +----------+
+       | Hoffman  |
+       | Glovanni |
+       | Liu      |
+       | Grass    |
+       | Clemens  |
+       | Pereira  |
+       | Cisneros |
+       +----------+
 
 7) Find the names and numbers of all salespeople who have more than one customer.
 ----------------------------------------------------------------------------------
-Query-
+Query- SELECT c.SNUM, s.SNAME, COUNT(*) AS COUNT FROM customers c, salespeople s WHERE c.SNUM = s.SNUM GROUP BY c.SNUM;
```

Comment:
A salesperson with more than 1 customer will only be shown.

Reply:
Done